mappings/net/minecraft/util/profiling/jfr/data/CpuLoad.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/unmapped/C_acuajsre net/minecraft/util/profiling/jfr/data/CpuLoad
+	FIELD f_cdlhimmg jvm D
```

Comment:
Do we want to consider making a new catergory for mapping record compontents? I think its unwise, but it might be useful when mapping, especially when record decompilation is improved, though it might not be hard to make an enigma plugin to find the method that returns the field with ASM and change it via that. @CheaterCodes

Reply:
It would be a good thing to have a special category for them.
It would avoid mistypes and it would make it clearer what's a record by reading the file.